app/Filters/PatientVerificationCodeFilters.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Filters;
+
+use Illuminate\Http\Request;
+
+class PatientVerificationCodeFilters extends QueryFilters
+{    
+    public function email() 
+    {
+        return $this->builder->where('deleted_at', null);
```

Comment:
When you use SoftDeletes trait in the model this condition applies by default

Reply:
done